flint.ui/src/components/Datepicker/DatepickerPoint.vue

```diff
@@ -1,101 +1,108 @@
+            <a-date-picker
+              v-model="endValue"
+              style="width: 100%"
+              :disabled-date="disabledEndDate"
+              placeholder="Select Date"
+              :open="endOpen"
+              @openChange="handleEndOpenChange"
+            />
+          </a-space>
+        </div>
+      </a-col>
+    </a-row>
 
-    <md-button class="md-primary">End Date</md-button>
-    <input v-modal="selectedendDate" type="date" class="datepicker-input" :class="[size]" />
-
-    <h3 class="text-xl font-bold mb-2 text-gray-600 justify-center">
-      Simulation length is
-      <span class="text-red-600">{{ date_diff > 0 ? date_diff.toFixed(2) + ' years' : 'invalid' }}</span>
-    </h3>
+    <div class="extra-bold">
+      <a-typography-title :level="3">
+        Simulation length is
+        <span style="color: #388591">{{
+          date_diff > 0 ? date_diff.toFixed(2) + ' years' : 'invalid'
+        }}</span></a-typography-title
+      >
+    </div>
   </div>
 </template>
 
 <script>
 import moment from 'moment'
 
 export default {
-  props: {
-    size: {
-      type: String,
-      default: 'medium',
-      validator(value) {
-        return ['small', 'medium', 'large'].includes(value)
-      }
+  data() {
+    return {
+      startValue: moment('1970-01-01').toString(),
+      endValue: moment('1971-01-01').toString(),
+      endOpen: false
     }
   },
   computed: {
     date_diff() {
-      let start_date = this.$store.state.point.config.LocalDomain.start_date.split('/')
-      console.log(start_date)
-      let end_date = this.$store.state.point.config.LocalDomain.end_date.split('/')
-      console.log(end_date)
-      //moment requires months to be zero indexed, hence subtracting 1
-      let start_year_month = moment([+start_date[0], +start_date[1] - 1])
-      let end_year_month = moment([+end_date[0], +end_date[1] - 1])
-
-      let date_difference = end_year_month.diff(start_year_month, 'years', true)
+      let date_difference = moment(this.endValue).diff(this.startValue, 'years', true)
       return date_difference
-    },
-    selectedstartDate: {
-      get() {
-        return this.$store.state.point.config.LocalDomain.start_date.split('/').join('-')
-      },
-      set(newValue) {
-        this.$store.commit('setNew_point_startDate', newValue.split('-').join('/'))
+    }
+  },
 
-        if (this.date_diff < 0) {
-          this.$toast.error('Start date should be less than end date', {
-            timeout: 5000
-          })
-        }
+  methods: {
+    disabledStartDate(startValue) {
+      const endValue = this.endValue
+      if (!startValue || !endValue) {
+        return false
       }
+      return startValue.valueOf() > endValue.valueOf()
     },
-
-    selectedendDate: {
-      get() {
-        return this.$store.state.point.config.LocalDomain.end_date.split('/').join('-')
-      },
-      set(newValue) {
-        this.$store.commit('setNew_point_endDate', newValue.split('-').join('/'))
```

Comment:
@joharikushagra You seem to have removed the logic for updating the store when a date is selected. Could you please add an explanation to that in the PR comments?